docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -1335,4 +1347,25 @@ The details have contents and a structure that is unique to each indicator.
 `state`::
 (string) The current state of the pipeline, including whether it is `loading`, `running`, `finished`, or `terminated`.
 ========
+`flow`::
+(object) A mapping of `<flow_metric>` to its rate values in various windows.
+
+.Available `<flow_metric>`-s:
+[%collapsible%open]
+==========
+`worker_utilization`::: A unitless metric indicating the percent of allocated worker-time that is being spent processing work (see: <<flow-pipeline-worker_utilization, Node Stats API metrics documentation>>).
+==========
+
+.Properties of each `<flow_metric>` are the value of the metric over various windows of time:
+[%collapsible%open]
+==========
+`current`::: the near-instantaneous _current_ rate
+`last_1_minute`::: the rate over the last minute (available when the pipeline has been running for at least one minute)
+`last_5_minutes`::: the rate over the last five minutes (available when the pipeline has been running for at least five minutes)
+`last_15_minutes`::: the rate over the last fifteen minutes (available when the pipeline has been running for at least fifteen minutes)
+`last_1_hour`::: the rate over the last hour (available when the pipeline has been running for at least one hour)
+`last_24_hours`::: the rate over the last twenty-four hours (available when the pipeline has been running for at least twenty-four hours)
+`lifetime`::: the _lifetime_ rate, since the pipeline was started
+==========
+
```

Comment:
```suggestion
`flow`::
(object) A mapping of `<flow_metric>` to its rate values in various windows.
+
.Available `<flow_metric>`-s:
[%collapsible%open]
==========
`worker_utilization`::: A unitless metric indicating the percent of allocated worker-time that is being spent processing work (see: <<flow-pipeline-worker_utilization, Node Stats API metrics documentation>>).
==========
+
.Properties of each `<flow_metric>` are the value of the metric over various windows of time:
[%collapsible%open]
==========
`current`::: the near-instantaneous _current_ rate
`last_1_minute`::: the rate over the last minute (available when the pipeline has been running for at least one minute)
`last_5_minutes`::: the rate over the last five minutes (available when the pipeline has been running for at least five minutes)
`last_15_minutes`::: the rate over the last fifteen minutes (available when the pipeline has been running for at least fifteen minutes)
`last_1_hour`::: the rate over the last hour (available when the pipeline has been running for at least one hour)
`last_24_hours`::: the rate over the last twenty-four hours (available when the pipeline has been running for at least twenty-four hours)
`lifetime`::: the _lifetime_ rate, since the pipeline was started
==========

```